server/public/styles.css

```diff
@@ -63,3 +74,14 @@ header {
   vertical-align: top;
   text-align: start;
 }
+
+.edit-button-container {
+  margin-top: 15px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
```

Comment:
Bootstrap has helpers for spacing, text, flex, and a whole bunch of other stuff. Please see how much custom CSS you can remove.

https://getbootstrap.com/docs/4.5/utilities/flex/
https://getbootstrap.com/docs/4.5/utilities/spacing/
https://getbootstrap.com/docs/4.5/utilities/text/
https://getbootstrap.com/docs/4.5/utilities/display/
https://getbootstrap.com/docs/4.5/utilities/sizing/